packages/flutter_tools/bin/fuchsia_builder.dart

```diff
@@ -3,13 +3,14 @@
 // found in the LICENSE file.
 
 import 'dart:async';
-import 'dart:io';
+import 'dart:io' as io;
```

Comment:
consider importing it as `import 'dart:io' as io hide File, FileSystemEntity;` ?

Or maybe have a base/io.dart file that imports dart:io, and the file system and process packages, and re-exports the supported APIs?

The idea being to stop people accidentally depending on the APIs we don't want to use directly any more.